### PR TITLE
rambox: 0.5.9 -> 0.5.10

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/franz/default.nix [hunk 1]
```diff
@@ -8,6 +8,17 @@ let
 
   version = "4.0.4";
 
+  runtimeDeps = [
```
Comment (NeQuissimus) [inline pkgs/applications/networking/instant-messengers/franz/default.nix]: This should probably be its own commit

Comment (gnidorah) [inline pkgs/applications/networking/instant-messengers/franz/default.nix]: @NeQuissimus Done, thanks!

**Comment by NeQuissimus** [comment]
LGTM, both apps build and run
